nvflare/app_common/ccwf/server_ctl.py

```diff
@@ -87,6 +88,7 @@ def __init__(
             result_clients: clients to receive final results
             max_status_report_interval:
             progress_timeout:
+            private_p2p: whether peer-to-peer communication should be private
```

Comment:
What does it mean for a peer-to-peer communication to be private VS public?

Reply:
I debated with myself what's the proper name for this arg, and decided on private_p2p. This means that when this flag is set, task communications between the two peers are encrypted. 

Private P2P communication is an additional level of protection on basic communication security (such as SSL). Each pair of peers have their own encryption keys to ensure that only they themselves can understand their messages, even if the messages may be relayed through other sites (e.g. server). Different pairs of peers have different keys. Currently, private P2P is enabled only when the system is in secure mode. This is because key exchange between peers requires both sides to have PKI certificates and keys, which requires the project to be provisioned in secure mode.

Reply:
Got it, maybe we could just add this explanation to the docstring as well?
Something like:
```
"""xxx

Args:
  xxx: yyy
  private_p2p: When this flag is set, task communications between the two peers are encrypted.

Note:
  Private P2P communication is an additional level of protection on basic communication security (such as SSL). 
  Each pair of peers have their own encryption keys to ensure that only they themselves can understand their messages, even if the messages may be relayed through other sites (e.g. server). 
  Different pairs of peers have different keys. Currently, private P2P is enabled only when the system is in secure mode. 
  This is because key exchange between peers requires both sides to have PKI certificates and keys, which requires the project to be provisioned in secure mode.

"""
```

Reply:
Yeah, I probably need to add docstrings for all args. They are all documented in the google doc, but it would be nice to include into docstrings.

Reply:
Added docstrings.